docs/developers/deploy-on-polygon.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Deploying the LSP on Polygon
-sidebar_label: Deploying the LSP on Polygon
+title: Deploying an LSP Contract
+sidebar_label: Deploying an LSP Contract
 ---
-UMA's infrastructure is also available on Polygon for developers to make use of as a scaling solution. The following will explain the details on how to deploy on Polygon. 
+This tutorial will show you how to deploy a Long Short Pair (LSP) contract by using the UMA `launch-lsp` repository. By following these instructions, you will be able to launch a custom LSP on Mumbai testnet, Polygon mainnet, Kovan testnet, and Ethereum mainnet.
 
-Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
+UMA's infrastructure is available on Polygon for developers to make use of as a scaling solution. Deploying to Polygon follows a near-identical launch process as to launching any UMA contract on Ethereum. Using one of our launch repo's and amending the destination network, you can quickly deploy your contracts on Polygon.
```

Comment:
Since this is a generalized tutorial now, it seems unnecessary to have something specifically for polygon. I'd recommend just removing this paragraph